app/models/post_flag.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class PostFlag < ActiveRecord::Base
```

Comment:
I think calling this `PostFlag` sets us down a dangerous road. We want to be able to generically define custom flags as admin, and then specify which things use those flags (e.g. post, topic, chat message). Granted this may have been done before we discussed in /t/127494/29 internally.

I think `CustomFlag` would be a better name, or even just `Flag` to be honest.